pkg/terraform/stages/ovirt/stages.go

```diff
@@ -0,0 +1,16 @@
+package ovirt
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/stages"
+	ovirttypes "github.com/openshift/installer/pkg/types/ovirt"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in oVirt.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage(ovirttypes.Name, "image", stages.WithNormalDestroy()),
```

Comment:
I know it is not for this PR but as a general note, I think that WithNormalDestroy and WithCustomDestroy are confusing.
From what I understand WithNormalDestroy means it will be destroyed during bootstrap, and not passing anything means that stage.destroyWithBootstrap is false and then it is destroyed on regular destroy.
I think you should change it to WithBootstrapDestroy and WithCustomBootstrapDestroy to make that clearer, its also not documented on the function

Reply:
https://github.com/openshift/installer/pull/5476